yrs/benches/benches.rs

```diff
@@ -357,7 +357,7 @@ where
             let (doc, _) = iter.next().unwrap();
             let mut txn = doc.transact();
             while let Some((_, update)) = iter.next() {
-                doc.apply_update_v1(&mut txn, update.as_slice());
```

Comment:
I removed `Doc::apply_update_v1` as it's redundant - it requires a transaction scope to work anyway, and Doc itself not really necessary for this method to work. It also leaves a room for ambiguity eg. `doc1.apply_update_v1(txn2, update)` where `txn2` was potentially created from different document . Making new transaction is trivial, so I think it's a better approach.